### PR TITLE
java runtime for java and kotlin functions

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

File: tests/Runtimes/RuntimesTest.php [hunk 1]
```diff
@@ -116,7 +126,7 @@ public function testSupportedRuntimes()
     {
         $this->assertNotEmpty($this->instance->get('node'));
         $this->assertNotEmpty($this->instance->getAll());
-        $this->assertCount(15, $this->instance->getAll(supported: false));
+        $this->assertCount(16, $this->instance->getAll(supported: false));
```
Comment (lohanidamodar) [inline tests/Runtimes/RuntimesTest.php:129]: @TorstenDittmann don't know what's happening here, but as I added new runtime, I incremented by 1 to pass the test :)

File: runtimes/java-11/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM openjdk:11-jre
```
Comment (TorstenDittmann) [inline runtimes/java-11/Dockerfile:1]: Is there an alpine version available?

File: tests/Runtimes/RuntimesTest.php (hunk 1 above)
Comment (TorstenDittmann) [reply tests/Runtimes/RuntimesTest.php:129]:
```suggestion
        $this->assertNotEmpty($this->instance->getAll(supported: false));
```

I think we should do this - so it is easier for us in the future 👍🏻

File: tests/Runtimes/RuntimesTest.php [hunk 3]
```diff
@@ -96,6 +96,16 @@ public function setUp(): void
                 'code' => $functionsDir . '/dotnet-5.0.tar.gz',
                 'command' => 'dotnet dotnet.dll',
                 'timeout' => 15,
+            ],
+            'java-11' => [
+                'code' => $functionsDir . '/java-11.tar.gz',
+                'command' => 'java HelloWorld',
+                'timeout' => 15,
+            ],
+            'kotlin-jar' => [
```
Comment (TorstenDittmann) [inline tests/Runtimes/RuntimesTest.php:105]: Right now this is not run.

I'll think of something so we can allow running multiple scripts on the same runtime.

File: runtimes/java-11/Dockerfile (hunk 2 above)
Comment (lohanidamodar) [reply runtimes/java-11/Dockerfile:1]: 11 doesn't have alpine version

File: tests/Runtimes/RuntimesTest.php (hunk 3 above)
Comment (lohanidamodar) [reply tests/Runtimes/RuntimesTest.php:105]: ok.

Comment (lohanidamodar) [reply tests/Runtimes/RuntimesTest.php:105]: may be restructure the array to specify the runtime to use to execute the test

Comment (TorstenDittmann) [reply tests/Runtimes/RuntimesTest.php:105]: Will do this once yours is merged, don't wanna create conflicts 👍🏻

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**